packages/amplify-category-api/package.json

```diff
@@ -75,8 +75,8 @@
     "@aws-amplify/amplify-provider-awscloudformation": "^8.8.0",
     "amplify-headless-interface": "^1.17.4",
     "amplify-util-headless-input": "^1.9.15",
-    "aws-cdk-lib": "^2.80.0",
-    "constructs": "^10.0.5"
+    "aws-cdk-lib": "^2.129.0",
```

Comment:
I see we're declaring major version compatible here, but pinning at the top level. Is that going to cause conflicts?